### DISCUSSION TITLE
Prisma migrate electron

### DISCUSSION THREAD

**Discussion by MitchellMonaghan** [prisma/prisma#6323] [Q&A]
For a electron app with a local db (sqlite) how would migrations work? Most apps like this I assume would want to migrate on app start. Currently migrations are executed through a cli command. Is it intended to include the cli into the build just so we can migrate programmatically? This seems wrong/odd to me or was not accounted for. Atm I wrote my own script that just consumes prisma's sql files using better-sqlite3.

**Comment by ryands17** [comment]
@MitchellMonaghan 👋 

Until we support programmatic migrations: https://github.com/prisma/prisma/issues/4703, you would need to run `prisma migrate` via exec if you need to do that in a script.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
